src/main/java/com/powsybl/network/conversion/server/NetworkConversionService.java

```diff
@@ -142,6 +142,17 @@ void importCaseAsynchronously(UUID caseUuid, String variantId, UUID reportUuid,
         notificationService.emitCaseImportStart(caseUuid, variantId, reportUuid, importParameters, receiver);
     }
 
+    Map<String, String> getDefaultImportParameters(UUID caseUuid) {
+        CaseInfos caseInfos = getCaseInfos(caseUuid);
+        Importer importer = Importer.find(caseInfos.getFormat());
+        Map<String, String> defaultValues = new HashMap<>();
+        importer.getParameters()
```

Comment:
Only `FUNCTIONAL` ?
If value is `null` keep `null` ?

```
        importer.getParameters()
                .stream()
                .forEach(parameter -> defaultValues.put(parameter.getName(), parameter.getDefaultValue() != null ? parameter.getDefaultValue().toString() : null));
```

Reply:
No, we also want to persist TECHNICAL parameters. I changed it.